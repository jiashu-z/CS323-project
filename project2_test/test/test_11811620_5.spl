struct A
{
  int num;
};

int test_11811620_5()
{
  int a;
  int b[10];
  struct A c;
  struct A d[10];

  a = a[0];
  a = b[9];
  c = c[10];
  c = d[9];
  a = b[a];
  a = b[c];
  return 0;
}