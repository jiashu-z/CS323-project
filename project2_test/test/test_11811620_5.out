Error type 10 at Line 13: indexing on non-array variable
Error type 5 at Line 13: unmatching type on both sides of assignment
Error type 10 at Line 15: indexing on non-array variable
Error type 5 at Line 15: unmatching type on both sides of assignment
Error type 12 at Line 18: indexing by non-integer
Error type 5 at Line 18: unmatching type on both sides of assignment
